Makefile

```diff
@@ -139,6 +139,23 @@ OPM = $(shell which opm)
 endif
 endif
 
+HELM = ./bin/helm
+HELM_VERSION = v3.15.0
+$(HELM):
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(HELM)) ;\
+	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+	wget -O helm.tar.gz https://get.helm.sh/helm-$(HELM_VERSION)-$${OS}-$${ARCH}.tar.gz ;\
```

Comment:
Verification steps failed to me due to missing `wget`. Can we use curl instead, since there are other commands that depend on it too?

```
$ make helm-install
bash: wget: command not found
make: *** [bin/helm] Error 127
```

Reply:
On a separate question, can we rename `charts/crds` to something like `charts/dependencies` or so?

cert-manager's manifests include a lot more than CRDs (namespaces, deployments, etc.) Additionally, authorino-operator's manifests (the one under `charts/templates`) also include CRDs, yet it's not in `charts/crds`.

Reply:
```suggestion
	curl -sL -o helm.tar.gz https://get.helm.sh/helm-$(HELM_VERSION)-$${OS}-$${ARCH}.tar.gz ;\
```